python/paddle/fluid/layers/nn.py

```diff
@@ -358,6 +360,12 @@ def dynamic_lstm(input,
         dtype(str): Data type. Choices = ["float32", "float64"], default "float32".
         name(str|None): A name for this layer(optional). If set None, the layer
                         will be named automatically.
+        h_0(Variable): The initial hidden state is an optional
+                       input. This is a tensor with shape (N x D), where N is the
+                       batch size and D is the hidden size.
+        c_0(Variable): The initial cell state is an optional
+                       input. This is a tensor with shape (N x D), where N is the
+                       batch size. `h_0` and `c_0` can be NULL but only at the same time.
```

Comment:
Tell the users if not set, the default is zero.

Reply:
Done.